src/Parallel/PhaseControlTags.hpp

```diff
@@ -9,14 +9,18 @@
 #include <utility>
 #include <vector>
 
-#include "Parallel/PhaseControl/PhaseChange.hpp"
 #include "Parallel/Reduction.hpp"
 #include "Parallel/Serialize.hpp"
-#include "ParallelAlgorithms/EventsAndTriggers/Trigger.hpp"
 #include "Utilities/Registration.hpp"
 #include "Utilities/TMPL.hpp"
 #include "Utilities/TaggedTuple.hpp"
 
+/// \cond
+class Trigger;
+template <typename PhaseChangeRegistrars>
+struct PhaseChange;
```

Comment:
When you only move the metafunctions to `Parallel` and keep this file in `PhaseControl` you can keep the dependency of `PhaseControl` on `EventsAndTriggers`. It might be appropriate to move `PhaseControl` to `ParallelAlgorithms/` at that point, but there's no need to do that in this PR. Just something for you to consider.